### PR TITLE
Add delete location bool param to PyTorchContainer load() call

### PR DESCRIPTION
**Pull request by akshjain83**
What?
Adds `delete_unzip_location_folder` param to `load()` method of `PyTorchSklearnContainer` to avoid implicit deletion of model artifact supplied to load() method. The changes in this PR are backward compatible.

Why?
Related issue: https://github.com/microsoft/hummingbird/issues/557

### CODE REVIEW

**Comment by akshjain83** [comment]
@interesaaat What would be the best next steps to propagate and distribute these changes once reviewed + approved? A patch?
I am looking for options that could unblock our production deployment. Please let me know, thanks!

**Comment by interesaaat** [comment]
> @interesaaat What would be the best next steps to propagate and distribute these changes once reviewed + approved? A patch? I am looking for options that could unblock our production deployment. Please let me know, thanks!

Yes we can push a new minor release of Hummingbird once this PR is merged (we have not pushed a new release in a while).

**Comment by akshjain83** [comment]
> > @interesaaat What would be the best next steps to propagate and distribute these changes once reviewed + approved? A patch? I am looking for options that could unblock our production deployment. Please let me know, thanks!
> 
> Yes we can push a new minor release of Hummingbird once this PR is merged (we have not pushed a new release in a while).

@interesaaat Perfect, what are the next steps for getting this PR merged? Please let me know if there is anything I can help with. Thanks.

**Comment by interesaaat** [comment]
@akshjain83 you need to sign the CLA otherwise merging is blocked. I am reviewing the PR right now.

**Review by interesaaat — APPROVED** [review]

**Comment by akshjain83** [comment]
@interesaaat I went ahead and signed the CLA. Please let me know if there is anything else I can help to roll this out. Thank you!

**Comment by ksaur** [comment]
@akshjain83 will push out v0.4.2 tomorrow!

**Comment by akshjain83** [comment]
> @akshjain83 will push out v0.4.2 tomorrow!

Thank you ❤️

**Comment by akshjain83** [comment]
@ksaur @interesaaat Could you please share when do you think we could expect 0.4.2 rolled out?

**Comment by ksaur** [comment]
It is in the works, and as promised will be done before i go home for the day.

**Comment by akshjain83** [comment]
> It is in the works, and as promised will be done before i go home for the day.

Thank you, really appreciate it. ❤️

**Pull request merged**
